concurrent/src/main/scala/scalaz/concurrent/Atomic.scala

```diff
@@ -4,6 +4,7 @@ package concurrent
 import effect._
 
 import java.util.concurrent.atomic.AtomicReference
+import java.util.function.UnaryOperator
```

Comment:
No, I think we should drop Java 7 in Scalaz 7.3. The reason for [`-target:jvm-1.7`](https://github.com/scalaz/scalaz/commit/bd41ba63a3f00c00617b4e1a776ff45842fa8dc0), Scala 2.10.6 does not accept `-target:jvm-1.8` and default is `-target:jvm-1.6`